app/assets/javascripts/controllers/container_dashboard/container_dashboard_utils.js

```diff
@@ -0,0 +1,116 @@
+angular.module('containerDashboard').factory('ContainerDashboardUtils', [function ContainerDashboardUtils () {
+  var createProvidersStatus = function() {
+    return {
+      title: "Providers",
+      count: 0,
+      href: "../ems_container/show_list",
+      notifications: []
```

Comment:
> Why are providers different here? (notification: {} vs notifications: [])

I believe its because the providers card is different in how the other cards only have ```warning``` and ```error``` notifications and the providers card has the option to add any sort of containers provider.
cc @jeff-phillips-18